tests/unit/httpd_test.cc

```diff
@@ -1690,6 +1690,18 @@ SEASTAR_TEST_CASE(case_insensitive_header) {
     return make_ready_future<>();
 }
 
+SEASTAR_TEST_CASE(broken_reply) {
+    http_response_parser parser;
+    parser.init();
+    char r101[] = "Lorem ipsum dolor sit amet, consectetur adipiscing elit.";
```

Comment:
The most strange usage of `Lorem ipsum` I have ever saw!

Reply:
Do you prefer "Quick brown fox..."? :)